rofs/rofs.go

```diff
@@ -20,19 +20,45 @@ var (
 	log = logging.MustGetLogger("rofs")
 )
 
-type filesystem struct {
-	pathfs.FileSystem
-	cache   string
+// Config represents a filesystem configuration object
+// Configuration objects can be used to manipulate some filesystem flags in runtime
+type Config struct {
 	store   meta.MetaStore
 	storage storage.Storage
+	cache   string
+}
+
+//SetMetaStore sets the filesystem meta store in runtime.
+func (c *Config) SetMetaStore(store meta.MetaStore) {
+	//TODO: should this be done atomically in a way that is synched ?
```

Comment:
I guess, from where do you read this attribute ?

Reply:
Well, to be honest i don't think we need to change how it's done atm. Code wise it should be safe. You can say that this todo is just a place mark if we have a problem in the future